src/main/webapp/app/exercises/text/assess/text-assessment-dashboard/text-assessment-dashboard.component.html

```diff
@@ -90,6 +90,7 @@ <h4 jhiTranslate="artemisApp.textAssessment.dashboard.heading" [translateValues]
                                     submission.id,
                                     'assessment'
                                 ]"
+                                [queryParams]="{ 'correction-round': submission.results?.length }"
```

Comment:
Just double-checking: should this line be the same as line 120: `[queryParams]="{ 'correction-round': submission.results!.length - 1 }"`?

Reply:
No, because on button is for assessing existing results, the other button is for non-existing results